### PR TITLE
Issue#111

### PR DESCRIPTION
**Pull request by bitbay**
## Issue

see #111 
## Description

Component extending another one doesn't need to redefine the base components outputs.
## Notes

Fixed.
Updated demo page with button to increase the items in PaginationDemo, Pager component should log on `(numPages)` event.

### CODE REVIEW

**Comment by bitbay** [comment]
shoot...looks like created the issue branch from the other branch, also being a pull-request for #110

File: tsconfig.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
     "experimentalDecorators": true,
     "noImplicitAny": true,
     "listFiles": false,
-    "noLib": false
+    "noLib": false,
+    "outDir": "./ng2-bootstrap"
```
Comment (valorkin) [inline tsconfig.json:14]: nope, this will not work as expected,
consumers of this lib will have to
`import 'ng2-bootstrap/ng2-bootstrap/ng2-bootstrap' to get to .js files

File: .gitignore [hunk 2]
```diff
@@ -15,13 +15,5 @@ npm-debug.log
...
-/demo/**/*.js
-/demo/**/*.js.map
-/demo/**/*.d.ts
-/components/**/*.js
-/components/**/*.js.map
-/components/**/*.d.ts
-ng2-bootstrap.js
-ng2-bootstrap.d.ts
-ng2-bootstrap.js.map
+/ng2-bootstrap
```
Comment (valorkin) [inline .gitignore:18]: please revert this changes
demo - is a gh-pages actually
and all other
in any situation we should not commit
generated files

File: components/pagination/pager.component.ts [hunk 3]
```diff
@@ -28,7 +28,6 @@ const PAGER_TEMPLATE = `
     'totalItems', 'itemsPerPage',
```
Comment (valorkin) [inline components/pagination/pager.component.ts:28]: input should be removed too

File: make.js [hunk 4]
```diff
@@ -32,7 +32,7 @@ async.waterfall([
 
 function getSystemJsBundleConfig(cb) {
   let config = {
-    baseURL: '..',
+    baseURL: '.',
```
Comment (valorkin) [inline make.js:35]: and, again, this is intentional
if not do this hack
npm and webpack users will be doing
`import 'ng2-bootstrap/ng2-bootstrap'`
and system.js users will do
`import 'ng2-bootstrap'`
but same code should work for both platforms

File: demo/components/pagination/pagination-demo.html [hunk 5]
```diff
@@ -10,11 +10,12 @@
                 (numPages)="smallnumPages = $event"></pagination>
     <pre class="card card-block card-header">The selected page no: {{currentPage}}/{{smallnumPages}}</pre>
     <button type="button" class="btn btn-info" (click)="setPage(3)">Set current page to: 3</button>
+    <button type="button" class="btn btn-info" (click)="increaseItems(3)">Increase items with: 3</button>
```
Comment (valorkin) [inline demo/components/pagination/pagination-demo.html:13]: this one is not obvious for end user
you need to press button several times to increase amount of pages
filling like pager is brokean

**Comment by valorkin** [comment]
fix included, thanks for your time :)
